example-config.json

```diff
@@ -66,7 +66,12 @@
         // Max number of requests to queue up before rejecting further requests.
         // Defaults to 0, which disables the limit.
         "max_queue_size": 500,
-        "admin_password": "xrp"
+        //if request contains header with authorization, clio will check if it matches this value's hash
+        //if matches, the request will be considered as admin request
+        "admin_password": "xrp",
+        //if local_admin is true, clio will consider requests come from 127.0.0.1 as admin requests
+        //it's true by default unless admin_password is set,'local_admin':true and 'admin_password' can not be set at the same time
+        "local_amdin": false
```

Comment:
Maybe there is no need to add one more config "local_admin". Because local_admin must not equal with admin_password existence